stream.go

```diff
@@ -113,7 +113,9 @@ type ClientStream interface {
 	// SendMsg is generally called by generated code. On error, SendMsg aborts
 	// the stream. If the error was generated by the client, the status is
 	// returned directly; otherwise, io.EOF is returned and the status of
-	// the stream may be discovered using RecvMsg.
+	// the stream may be discovered using RecvMsg. However, if
+	// StreamDesc.ClientStreams was set to false, then nil error is returned
```

Comment:
How about this instead for the last sentence:

```
For unary or server-streaming RPCs (StreamDesc.ClientStreams is false),
a nil error is returned unconditionally.
```

Reply:
done please review